docs/exts/docs_build/code_utils.py

```diff
@@ -30,7 +30,7 @@
 DOCKER_AIRFLOW_DIR = os.path.join(DOCKER_PROJECT_DIR, "/airflow")
 ALL_PROVIDER_YAMLS = load_package_data()
 AIRFLOW_SITE_DIR = os.environ.get('AIRFLOW_SITE_DIRECTORY')
-PROCESS_TIMEOUT = 4 * 60
+PROCESS_TIMEOUT = 8 * 60  # 400 seconds
```

Comment:
8*60 = 400? Isn't it just 8 mins?

Reply:
```suggestion
PROCESS_TIMEOUT = 8 * 60
```

Reply:
yeah.. I changed/commented out and did not remove it :)

Reply:
8*80 = 480

Reply:
8*80 = 640 ;) . Yeah. I removed the comment as it would be all to easy to get it out of sync as it was there.